SpatialGDK/Extras/git/post-merge

```diff
@@ -6,8 +6,8 @@ set -e -u -o pipefail
 changed_files="$(git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD)"
 
 [[ "$(uname -s)" != "Darwin" ]] \
-    && setup_command="exec \"cmd.exe /c Setup.bat SETUP_ARGS\"" \
+    && setup_command="exec cmd.exe /c Setup.bat SETUP_ARGS" \
     || setup_command="sh Setup.sh SETUP_ARGS"
 
-echo "${changed_files}" | grep --quiet RequireSetup && setup_command
+echo "${changed_files}" | grep --quiet RequireSetup && $setup_command
```

Comment:
```suggestion
echo "${changed_files}" | grep --quiet RequireSetup && ${setup_command}
```
I think is preferred, but I am not even sure if that will work. Batch files are magic.

Reply:
I've tried locally and it seems to work. `$x` and `${x}` are the same for the most part, the braces do make it clearer when the variable name ends (consider `echo $keyvalue` vs `echo ${key}value`)

Reply:
Yeah, that's what I thought as well.
Just to keep the consistency with `${changed_files}`, which also do not need it, but it's nice to have the strict version everywhere.